cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -25,6 +25,16 @@ describe('Top toolbar tests.', function() {
 			.should('exist');
 	});
 
+	it('Apply underline.', function() {
+		cy.get('#tb_editbar_item_underline')
+			.click();
+
+		helper.selectAllText(false);
+
+		cy.get('#copy-paste-container p')
```

Comment:
Instead of just checking 'p' here, which means we have a paragraph here, you should check whether this paragraph has an underline property. Check 'Apply underline.' test in https://github.com/CollaboraOnline/online/blob/master/cypress_test/integration_tests/mobile/writer/apply_font_spec.js